Documentation/CRDs/Cluster/external-cluster/upgrade-external.md

```diff
@@ -0,0 +1,59 @@
+# External Cluster Upgrades
+
+Upgrading external cluster is little tricky, We need to upgrade the Ceph cluster and Rook cluster seperalty with their new versions, Plus upgrading the versions we also need to update the external provider cluster connection to consumer Rook cluster.
+
+To upgrade the provider ceph cluster to use the latest ceph user permissions, which will be required for the rook reconciliation, based on the new implementation added to the Rook.
```

Comment:
Does my suggestion above also cover this concept of new permissions and properties? Then no need for this paragraph.
```suggestion
```